fluentd/Dockerfile

```diff
@@ -16,23 +15,28 @@ LABEL io.k8s.description="Fluentd container for collecting of docker container l
 
 # activesupport version 5.x requires ruby 2.2
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
-  yum install -y --setopt=tsflags=nodocs \
-  gcc-c++ \
-  ruby \
-  ruby-devel \
-  libcurl-devel \
-  make && \
-  mkdir -p ${HOME} && \
-  gem install --no-rdoc --no-ri fluentd:${FLUENTD_VERSION} 'activesupport:<5' fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash && \
-  mkdir -p ${THROTTLE_CONF_LOCATION} && \
-  touch ${THROTTLE_CONF_LOCATION}/settings
+    yum install -y --setopt=tsflags=nodocs \
+      gcc-c++ \
+      ruby \
+      ruby-devel \
+      libcurl-devel \
+      make && \
+    yum clean all
+RUN mkdir -p ${HOME} && \
```

Comment:
Any particular reason? I wanted to put the yum stuff in a separate layer so
it would cache even if the stuff after it changed.

On Tue, Jul 12, 2016 at 4:12 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In fluentd/Dockerfile
> <https://github.com/openshift/origin-aggregated-logging/pull/195#discussion_r70512802>
> :
>
> > -  ruby \
> > -  ruby-devel \
> > -  libcurl-devel \
> > -  make && \
> > -  mkdir -p ${HOME} && \
> > -  gem install --no-rdoc --no-ri fluentd:${FLUENTD_VERSION} 'activesupport:<5' fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash && \
> > -  mkdir -p ${THROTTLE_CONF_LOCATION} && \
> > -  touch ${THROTTLE_CONF_LOCATION}/settings
> > +    yum install -y --setopt=tsflags=nodocs \
> > +      gcc-c++ \
> > +      ruby \
> > +      ruby-devel \
> > +      libcurl-devel \
> > +      make && \
> > +    yum clean all
> > +RUN mkdir -p ${HOME} && \
>
> should probably remove this RUN
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/195/files/c79a4f8ff336fad4187dcb823b73638724df6f84#r70512802>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-p8QavFU4wglzZaSxniy70_xmXa7ks5qU_UpgaJpZM4JHWV8>
> .
>